test-kitchen/skills/cookoff/SKILL.md

```
@@ -221,30 +222,32 @@ Checking: security, logic errors, edge cases
 Fresh-eyes complete: 1 minor issue
 ```
 
-**Step 4: Compare survivors**
+**Step 4: Invoke Judge Skill**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)